### PR TITLE
DAMD-65: Retrospective approval of 2D pipeline-related changes

### PR DESCRIPTION
**Pull request by PaulPrice**

### CODE REVIEW

File: datamodel.txt [hunk 1]
```diff
@@ -288,12 +290,13 @@ keywords in the header to disambiguate this.
...
-HDU #4 WAVELENGTH  Wavelength solution in nm (vacuum)	       [FLOAT] 	      NROW*NFIBER
-HDU #5 SKY         Sky flux in same units as HDU1    	       [FLOAT]	      NROW*NFIBER
-HDU #6 CONFIG      Information about the PFI, targetting, etc. [BINARY FITS TABLE]
+HDU #1 FIBERID      Fiber identifier                        [32-bit INT]    NROW*NFIBER
+HDU #2 WAVELENGTH   Wavelength (nm, vacuum)                 [FLOAT]         NROW*NFIBER
+HDU #3 FLUX         Flux (nJy)                              [FLOAT]         NROW*NFIBER
+HDU #4 MASK         Pixel mask                              [32-bit INT]    NROW*NFIBER
+HDU #5 SKY          Sky flux (nJy)                          [FLOAT]         NROW*NFIBER
+HDU #6 COVAR        Near-diagonal part of flux covariance   [FLOAT]         NROW*3*NFIBER
+HDU #7 IDENTITY     Keyword-value pairs identifying data    [BINARY TABLE]
```
Comment (hassanxp) [inline datamodel.txt]: Following a quick review with RHL: The original name `CONFIG` makes more sense. The new proposed name `IDENTITY` sounds like a identity matrix. Can we keep the original name?

File: datamodel.txt [hunk 2]
```diff
@@ -518,15 +521,15 @@ HDU#4 ZLINES                    Binary table            [FITS BINARY TABLE]
  pfsDesign-0xad349fe21234abcd.fits
 	The fiber and targetting information for the PFI configuration with hash 0xad349fe21234abcd
 
- pfsFlat-000333-m2.fits
-	Flat for spectrograph 2, medium resolution arm, valid for 333 <= visit < ??
- pfsBias-000333-r1.fits
+ pfsFlat-2019-08-29-000333-m2.fits
```
Comment (hassanxp) [inline datamodel.txt]: Using date stamps may not have the necessary resolution. Can we use `visit0` instead? Failing that a more accurate date stamp (to the minute)?

Comment (PaulPrice) [reply datamodel.txt]: The proposal uses _both_ date and `visit0`. I don't think there's a reason we need choose only one.

File: datamodel.txt (hunk 1 above)
Comment (hassanxp) [reply datamodel.txt]: Seems that you have reverted to the original name of `CONFIG`? Perfect thanks.

**Pull request merged**